enhancements/microshift/kubernetes-for-industrial-edge.md

```diff
@@ -0,0 +1,433 @@
+---
+title: kubernetes-for-industrial-edge
+authors:
+  - dhellmann
+reviewers:
+  - "@copejon, MicroShift contributor"
+  - "@deads2k, control plane expert"
+  - "@fzdarsky, MicroShift architect"
+  - "@ggiguash, MicroShift contributor"
+  - "@jogeo, MicroShift contributor"
+  - "@mangelajo, MicroShift contributor"
+  - "@oglok, MicroShift contributor"
+  - "@sallyom, MicroShift contributor"
```

Comment:
@mfojtik @tkashem @stlaz @ingvagabund @hasbro17 Please let me know which of your team members to include as reviewers here.

Reply:
moving this thread to the renamed file